tuf/util.py

```diff
@@ -898,6 +901,42 @@ def load_json_string(data):
 
 
 
+
+
+
+def load_der_string(data):
+  """
+  <Purpose>
+    Deserialize 'data' (DER string) to a Python object. This supports only
+    what tuf.asn1_codec supports, which at the time of this writing is
+    signed role metadata (timestamp, snapshot, root, or targets) converted into
+    ASN.1 and then encoded as DER.
+
+  <Arguments>
+    data:
+      A DER string, as would be output by e.g.
+      asn1_codec.convert_signed_metadata_to_der()
+
+  <Exceptions>
+    tuf.Error, if 'data' cannot be deserialized to a Python object.
+
+  <Side Effects>
+    None.
+
+  <Returns>
+    Deserialized object.  For example, a dictionary.
```

Comment:
1. You're right that the <Returns> doc should be better. In the next round of pushes, it'll be:
```python
    """
    Deserialized object, conforming to tuf.formats.SIGNABLE_SCHEMA.
    The signatures contained in the returned object, if any, will have been
    unchanged. If, for example, the signatures were over a DER object,
    they will remain that way, even though the 'signed' portion will no longer
    be in DER.
    """
```
"convert_signed_der_to_dersigned_json" is meant to indicate that the Python dictionary returned retains a signature over the DER format ("dersigned"). Perhaps it's too ambitious a name. :P